doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,661 @@
+
+* :ref:`matter_smoke_co_alarm_sample` sample:
+
+  * Added:
+
+    * Support for ICD dynamic SIT LIT switching (DSLS).
+    * Support for the ``nrf54l15dk/nrf54l10/cpuapp`` board target.
+
+Peripheral samples
+------------------
+
+* Added support for the ``nrf54l15dk/nrf54l05/cpuapp`` and ``nrf54l15dk/nrf54l10/cpuapp`` board targets in the :ref:`radio_test` sample.
+
+Protocol serialization samples
+------------------------------
+
+* Updated GPIO pins on the nRF54L15 DK used for communication between the client and server over UART.
+  One of the previously selected pins was also used to drive an LED, which may have disrupted the UART communication.
+
+Thread samples
+--------------
+
+* Removed support for the ``nrf5340dk/nrf5340/cpuapp/ns`` board target for all samples.
+
+* :ref:`ot_cli_sample` sample:
+
+  * Added support for the ``nrf54l15dk/nrf54l10/cpuapp`` board target.
+
+* :ref:`ot_coprocessor_sample` sample:
+
+  * Added support for the ``nrf54l15dk/nrf54l10/cpuapp`` and ``nrf54l15dk/nrf54l05/cpuapp`` board targets.
+
+Other samples
+-------------
+
+* :ref:`coremark_sample` sample:
+
+  * Updated:
+
+    * Configuration for the :ref:`zephyr:nrf54h20dk_nrf54h20` board to support multi-domain logging using the ARM Coresight STM.
+    * The logging format in the standard logging mode to align it with the format used in the multi-domain logging mode.
+    * Support for alternative configurations to use the :ref:`file suffix feature from Zephyr <app_build_file_suffixes>`.
+      The following file suffixes are supported as alternative configurations:
+
+      * ``flash_and_run``
+      * ``heap_memory``
+      * ``static_memory``
+      * ``multiple_threads``
+
+Drivers
+=======
+
+This section provides detailed lists of changes by :ref:`driver <drivers>`.
+
+Wi-Fi drivers
+-------------
+
+* Added the :ref:`nrf70_wifi_tx_power_calculation` section to the :ref:`nrf70_wifi` page.
+
+Libraries
+=========
+
+This section provides detailed lists of changes by :ref:`library <libraries>`.
+
+Binary libraries
+----------------
+
+* :ref:`liblwm2m_carrier_readme` library:
+
+  * Updated the :ref:`req_appln_limitations` page to clarify carrier-specific requirements.
+    Added overlay files and documentation to :ref:`serial_lte_modem` application and :ref:`lwm2m_carrier` sample to guide in the correct usage of LwM2M carrier library for SoftBank and LG U+.
+
+Bluetooth libraries and services
+--------------------------------
+
+* Added the :ref:`rreq_readme` and :ref:`rrsp_readme` libraries.
+
+* :ref:`hogp_readme` library:
+
+  * Updated the :c:func:`bt_hogp_rep_read` function to forward the GATT read error code through the registered user callback.
+    This ensures that API user is aware of the error.
+
+* :ref:`bt_fast_pair_readme` library:
+
+  * Added support in build system for devices that do not support the :ref:`partition_manager`.
```

Comment:
Hi @peknis, please move this entry back to the changelog, it is relevant for 2.9-nRF54H20.